bonobo/settings.py

```diff
@@ -93,7 +90,7 @@ def clear(self):
     "LOGGING_LEVEL",
     formatter=logging._checkLevel,
     validator=logging._checkLevel,
-    default=lambda: logging.DEBUG if DEBUG.get() else logging.INFO,
+    default=logging.INFO,
```

Comment:
thoughts on making `DEBUG` the default level? im a bit concerned we'll do something like blow out our datadog caps if we set the default as `INFO` (iirc we don't ship `DEBUG` logs to ddog).

Reply:
yeah the killer here is ALPR; i think we'll be logging INFO on every alpr run

Reply:
in prod DEBUG = false so the default was already INFO.

 this logging level is not the level at which logs are logged, its the level at which the logger prints the logs https://docs.python.org/3/library/logging.html#logging.Logger.setLevel